utils/cipher.h

```diff
@@ -35,6 +35,20 @@ typedef struct ta_cipher_ctx {
   const char* device_id;   /**< Device id */
 } ta_cipher_ctx;
 
+/**
+ * @brief Initialize logger
```

Comment:
The description is too generic. You shall mention cipher.

Reply:
Done.